lib/tasks/data/text_similarity_data.rake

```diff
@@ -157,10 +158,21 @@ namespace :check do
         puts 'Please provide the AWS credentials.'
         exit 1
       end
+      body = data.collect{ |row| row.to_json }.join("\n")
+      key = "text-similarity-data-export-#{Time.now.strftime('%Y-%m-%d')}.json.zip"
```

Comment:
For clarity/consistency, please change the extension to `.gz`. `Zlib::GzipWriter` will generate the output in Gzip format, not Zip.

Reply:
Done